xtask/src/swift.rs

```diff
@@ -242,3 +237,18 @@ fn build_xcframework(
 
     Ok(())
 }
+
+fn move_swift_files(source: &Utf8PathBuf, destination: &Utf8PathBuf) -> Result<()> {
+    source.read_dir_utf8()?.for_each(|entry| {
+        let Ok(entry) = entry else { return };
+
+        if entry.file_type().map(|t| t.is_file()).is_ok_and(|f| f) {
+            let path = entry.path();
+            if path.extension() == Some("swift") {
+                let file_name = path.file_name().expect("Failed to get file name");
+                rename(path, destination.join(file_name)).expect("Failed to move swift file");
+            }
+        }
+    });
```

Comment:
```suggestion
    for entry in source.read_dir_utf8()? {
        let entry = entry?;

        if entry.file_type()?.is_file() {
            let path = entry.path();
            if path.extension() == Some("swift") {
                let file_name = path.file_name().expect("Failed to get file name");
                rename(path, destination.join(file_name)).expect("Failed to move swift file");
            }
        }
    }
```

Reply:
Ok makes sense. I wasn't 100% sure if these errors should fail the whole build or just be ignored. Updated 👍